### PR TITLE
Use gender-neutral language

### PR DESCRIPTION
**Pull request by pcbro**
Fixes inconsiderate comment

### CODE REVIEW

File: lib/Sema/TypeCheckDecl.cpp [hunk 1]
```diff
@@ -2930,7 +2930,7 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
 
     if (!IsSecondPass) {
       for (unsigned i = 0, e = PBD->getNumPatternEntries(); i != e; ++i) {
-        // Type check each VarDecl in that his PatternBinding handles.
+        // Type check each VarDecl in that their PatternBinding handles.
```
Comment (cwillmor) [inline lib/Sema/TypeCheckDecl.cpp]: I think this is supposed to say "Type check each VarDecl **that this** PatternBinding handles.".

Comment (jeeftor) [inline lib/Sema/TypeCheckDecl.cpp]: @cwillmor  good point.  I think this is a bad PR

Comment (tangentfairy) [inline lib/Sema/TypeCheckDecl.cpp]: I think this is hilarious but @cwillmor is right and it is just a typo. We have to appreciate @pcbro's sense of humour though :stuck_out_tongue:

Comment (Shirk) [inline lib/Sema/TypeCheckDecl.cpp]: @tangentfairy - I'm not sure it's humor at all.. look here #165

**Comment by peterkellyonline** [comment]
get a grip people

**Comment by James-Parsons** [comment]
@peterkellyonline  seriously, I bet the developers are a bunch of misogynist pigs :P. If you get offended over this, you need to grow up.

**Comment by bprzemyslaw** [comment]
It appears to me somebody just wants to become a contributor really bad.

**Comment by jckarter** [comment]
I think @cwillmor's right that the typo is different. Eliminating pointless gendered language in general is welcome though. (Comments like "get a grip", "this doesn't matter", and other silencing tactics are disrespectful and definitely *aren't* welcome.)

**Comment by tangentfairy** [comment]
Completely agree with you @jckarter :smile:

**Comment by AdamHess** [comment]
@pcbro is just being a troll

**Comment by benlangmuir** [comment]
This was fixed per @cwillmor's suggestion as part of #186.

**Comment by jckarter** [comment]
@AdamHess Remember Poe's law. If their contribution is helpful and aligned with our goals, who's really getting trolled?

File: lib/Sema/TypeCheckDecl.cpp (hunk 1 above)
Comment (jcfrancisco) [inline lib/Sema/TypeCheckDecl.cpp]: @Shirk The user @pcbro is definitely trying to get a rise out of someone -- the account is fashioned after the character "PC Principal" in South Park, who's somewhat of a caricature of an politically correct person. This doesn't mean the PRs should be ignored, but the intent behind them is pretty clear to me.

**Comment by AdamHess** [comment]
@jckarter this is not @pcbro first pull request to this repo. Additionally, it's **NOT** helpful it's a typo. @pcbro is not being helpful by pointing out typos but making uneducated fun of various programming terminology and practices see earlier pull request: https://github.com/apple/swift/pull/165  I cant beleive that the project maintainers even merged the request.

**Comment by AdamHess** [comment]
God I just realized @jckarter is working in cahoots with @pcbro ... so its basically apple trolling the dev community. Good job.

**Comment by hirad** [comment]
@AdamHess @pcbro's entire account is referencing a South Park parody of SJWs. I think you're right. I don't understand why he's taken seriously.

**Comment by steveklabnik** [comment]
@jckarter exactly, :+1:

**Comment by AdamHess** [comment]
@hirad I'm well aware of it. The problem is that his pull requests are being excepted and merged into the master branch.

**Comment by jckarter** [comment]
I think we're done here.

**Pull request closed without merging**